public/app/percona/dbaas/components/Kubernetes/OperatorStatusItem/KubernetesOperatorStatus/KubernetesOperatorStatus.tsx

```diff
@@ -1,15 +1,34 @@
 import React, { FC } from 'react';
-import { useStyles } from '@grafana/ui';
+import { Button, useStyles } from '@grafana/ui';
+import { OPERATOR_LABELS } from 'app/percona/shared/core';
+import { Messages } from 'app/percona/dbaas/DBaaS.messages';
 import { getStyles } from './KubernetesOperatorStatus.styles';
 import { KubernetesOperatorStatus as Status, KubernetesOperatorStatusProps } from './KubernetesOperatorStatus.types';
 import { getStatusLink } from './KubernetesOperatorStatus.utils';
 import { OperatorStatus } from './OperatorStatus/OperatorStatus';
+import { DATABASE_COMPONENT_TO_UPDATE_MAP } from '../../Kubernetes.constants';
 
-export const KubernetesOperatorStatus: FC<KubernetesOperatorStatusProps> = ({ operator, databaseType }) => {
+export const KubernetesOperatorStatus: FC<KubernetesOperatorStatusProps> = ({
+  operator,
+  databaseType,
+  kubernetes,
+  setSelectedCluster,
+  setOperatorToUpdate,
+  setUpdateOperatorModalVisible,
+}) => {
   const styles = useStyles(getStyles);
   const { status, availableVersion } = operator;
-  const showLink =
-    status === Status.unavailable || ((status === Status.ok || status === Status.unsupported) && !!availableVersion);
+  const isVersionAvailable = (status === Status.ok || status === Status.unsupported) && !!availableVersion;
+  const showLink = status === Status.unavailable || isVersionAvailable;
+  const updateOperator = () => {
+    setSelectedCluster(kubernetes);
+    setOperatorToUpdate({
+      operatorType: DATABASE_COMPONENT_TO_UPDATE_MAP[databaseType],
+      operatorTypeLabel: OPERATOR_LABELS[databaseType],
+      ...operator,
```

Comment:
shouldn't the ellipsis come before, as you are overwriting those two props?

Reply:
The operator doesn't have `operatorType` or `operatorTypeLabel` attributes. If it had we wouldn't need to create a new object here. Also in the unlikely scenario that it gets added to the operator in the future is fine to override since the data is consistent.

Reply:
Got it 👍